IbisHardwareIGSIO/ibishardwareIGSIO.cpp

```diff
@@ -392,38 +392,20 @@ void IbisHardwareIGSIO::InitPlugin()
 
     // Look for the Plus Toolkit path and Plus Server executable
     m_plusServerExec = GetIbisAPI()->GetCustomPath( PlusServerExecutable );
-    bool ok = true;
+    bool ok = false;
     if( m_plusServerExec != QString::null && !m_plusServerExec.isEmpty() )
     {
         QFileInfo fi( m_plusServerExec );
         if( !( fi.isFile() && fi.isExecutable() ) )
-            ok = false;
-    }
-    else
-    {
-        m_plusToolkitPathsFilename = m_plusConfigDirectory + QString("plus-toolkit-paths.txt");
-        if( QFile::exists( m_plusToolkitPathsFilename ) )
         {
-            QFile pathFile( m_plusToolkitPathsFilename );
-            pathFile.open( QIODevice::ReadOnly | QIODevice::Text );
-            QTextStream pathIn( &pathFile );
-            pathIn >> m_plusServerExec;
-            QFileInfo fi( m_plusServerExec );
-            if( fi.isFile() && fi.isExecutable() )
-                GetIbisAPI()->RegisterCustomPath( PlusServerExecutable, fi.absoluteFilePath() );
-            else
-                ok = false;
+            GetIbisAPI()->UnRegisterCustomPath( PlusServerExecutable );
```

Comment:
You should never unregister the path. Otherwise, it won't appear in the preference dialog and the user won't be able to set the path.